src/components/TalkToUs/LeaveMessage/index.js

```diff
@@ -141,7 +144,12 @@ const Callback = () => {
           </p>
         ) : null}
         <Button type="submit" css={submitButton} disabled={disabled}>
```

Comment:
When `isLoading` is true, the button content will "jump". We need to avoid this behaviour. Please
1. fix the button size (width: 12rem, height: 4rem)
2. render like this: `isLoading ? <ClipLoader /> : "SUBMIT"`

Reply:
Fixed.